README.md

```diff
@@ -367,6 +369,16 @@ Each of these options can be overridden via command-line options (see [below](#c
 
 A typical configuration for a CI environment will enable all forms of caching, whereas for local development you may want to set `write_remote_cache: false` to avoid waiting for remote cache writes.
 
+### Docker CLI
+
+While still experimental, one change out the CLI command used by Toast. At this time only drop-in replacements for the Docker CLI, such as Podman may be used.
```

Comment:
Improper comma use. Either add a comma after `Podman` or remove the one after `CLI`.

I would prefer to explain this as (paraphrased):

- You can specify the Docker CLI binary, which by default is `docker`.
- Toast will search your `PATH` for the specified binary.
- You can also use this mechanism to switch to a drop-in replacement for the Docker CLI, such as Podman.